src/otable/ocolumn.py

```diff
@@ -1,7 +1,20 @@
 """Table data types backed by objects."""
 
 import collections.abc
-from typing import Iterable, Sequence, TypeVar, cast, overload
+from operator import attrgetter
+from typing import Any, Callable, Iterable, Sequence, TypeVar, cast, overload
+
+__all__ = ['OColumn']
```

Comment:
I'm not a fan of `from ... import *`, so to me `__all__` is little more than documentation of what's meant to be public. For that, in general I prefer using the `_` prefix for the non-public members, because IMO it helps when reading code, more than a single line at the top.